modules/persistenceService/src/main/resources/db/migration/V0__CREATE_USER_TABLE.sql

```diff
@@ -0,0 +1,4 @@
+create table user(
+    id serial PRIMARY KEY,
```

Comment:
Not a big deal, but I prefer `uuid` and have control on the generation in our service

Reply:
Sure, I will make the change to `uuid`